control/filehash.go

```diff
@@ -66,8 +66,8 @@ func (d *DebianFileHash) Validate() (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	if size := stat.Size(); size != int64(d.Size) {
-		return false, fmt.Errorf("Error! Size mismatch! %d != %d", size, d.Size)
```

Comment:
btw, I switched the message here to not include `Error` because that part of the text is already implied by the type, so usually this is displayed with something like `"error doing %s: %v\n", "some-task", err`, so the extra `Error!` is kind of silly :smile: